easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb

```diff
@@ -7,7 +7,7 @@
 
 easyblock = 'PackedBinary'
 
-name = 'sratoolkit'
+name = 'SRA-Toolkit'
 version = '2.5.7'
```

Comment:
Travis is complaining about a discrepancy with the expected filename:
```
 Filename '/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.5.7-centos_linux64.eb' of parsed easyconfig matches expected filename 'SRA-Toolkit-2.5.7.eb'
```

and

```
Filename '/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.8.2-1-centos_linux64.eb' of parsed easyconfig matches expected filename 'SRA-Toolkit-2.8.2-1.eb'
```

It's expecting `centos_linux64` to be the `versionsuffix`?

Reply:
Indeed, I overlooked that, fixed in 28d19f8